internal/pkg/describe/lb_web_service.go

```diff
@@ -289,36 +291,75 @@ type WebServiceRoute struct {
 	URL         string `json:"url"`
 }
 
-// ServiceDiscovery contains serialized service discovery info for an service.
-type ServiceDiscovery struct {
+type internalEndpoint struct {
 	Environment []string `json:"environment"`
-	Namespace   string   `json:"namespace"`
+	Endpoint    string   `json:"endpoint"`
 }
 
-type serviceDiscoveries []*ServiceDiscovery
+type internalEndpoints []*internalEndpoint
+
+type serviceDiscoveries internalEndpoints
 
-func (s serviceDiscoveries) humanString(w io.Writer) {
-	headers := []string{"Environment", "Namespace"}
+func (sds *serviceDiscoveries) collectEndpoints(descr envDescriber, svc, env, port string) error {
```

Comment:
nit: these can probably be moved to `service.go` since they aren't lbws specific.